.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30  # default is 360
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install Dependencies
+      run: |
+        pip install pipenv
+        pipenv install --ignore-pipfile --deploy --system --dev
+    - name: Generate protobuf files
+      run: make protos
+    - name: Install extra dependencies
+      run: |
+        sudo apt-get install -y graphviz librocksdb-dev libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev
+        pip install "cython<0.30"
+        pip install python-rocksdb==0.7.0
```

Comment:
Are these 2 lines needed after the MR to update the Dockerfile (https://gitlab.com/HathorNetwork/hathor-python/-/merge_requests/411/diffs)?

Reply:
Yes, they are. The Github workflow doesn't use the Dockerfile. These 2 lines will be needed whenever we want to install rocksdb.

It may be able to fix this with Poetry. At the very least in Poetry we can have "extra" dependencies (just like on setup.py) and move python-rocksdb there, still I'm not sure if it iis  possible to force Cython to be installed before python-rocksdb with it.

Reply:
Ok, got it.